adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformAlignments.scala

```diff
@@ -385,6 +388,10 @@ class TransformAlignments(protected val args: TransformAlignmentsArgs) extends B
 
     val stringencyOpt = Option(args.stringency).map(ValidationStringency.valueOf(_))
 
+    if (args.maxReadLength > 0) {
```

Comment:
It may make more sense to set the default args.maxReadLength to 10,000 and remove the if statement.

Reply:
If it is set to the default here in the cli class, then it is always set here, and will clobber the configuration property if defined on the command line or elsewhere.